### DISCUSSION TITLE
Set column of type chrono DateTime in ActiveModel set method

### DISCUSSION THREAD

**Discussion by skopz356** [SeaQL/sea-orm#1380] [Q&A] [answered]
Hi guys, when I try to set chrono Local::now() using ActiveModel::set I get error:
 `Result::unwrap() on an Err value: ValueTypeErr /home/tomas/.cargo/registry/src/github.com-1ecc6299db9ec823/sea-query-0.27.2/src/value.rs:218:27. `

I have defined the field in model as follows:
`pub last_changed_date: Option<DateTimeUtc>,`

`let mut new_customer : customer::ActiveModel = Default::default();`
`new_customer.set(customer::Column::LastChangedDate, Local::now().into());`

Am I doing something wrong? Thank you

**Comment by billy1624** [comment, marked as answer]
Hey @skopz356, I saw your `last_changed_date` field is of type `Option<DateTimeUtc>` that means you should provide a [`Utc::now()`](https://docs.rs/chrono/latest/chrono/offset/struct.Utc.html) instead of `Local::now()` to the corresponding field in the active model.

**Reply by skopz356** [reply]
@billy1624 Thank you so much! I do not know how could I miss that :/

**Reply by billy1624** [reply]
No worries :)
